.github/workflows/terraform-fmt.yml

```diff
@@ -0,0 +1,17 @@
+name: Terraform Format
+
+on:
+  push:
+    paths:
+      - '**.tf'
+  pull_request:
+    paths:
+      - '**.tf'
+
+jobs:
+  terraform-fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hashicorp/setup-terraform@v2
+      - run: find . -name '*.tf' -exec terraform fmt {} \;; git diff --exit-code
```

Comment:
**suggstion**: I think we can avoid git diff by using `terraform fmt -check -diff`. We can probably avoid `find` too by using the `-recursive` flag

Reply:
Ah, good find.